packages/functions/src/dal/migrationScripts/ChangeAppointmentStatusMigration.ts

```diff
@@ -0,0 +1,39 @@
+import { getAllLatestAppointments, setAppointment } from "../AppointmentDataAccessLayer";
+import { AppointmentStatus } from "@zm-blood-components/common/src";
+const fs = require('fs');
+
+export async function addStatusForAppointments(){
+  const appointments = await getAllLatestAppointments();
```

Comment:
Why latest? Why not all?

Reply:
just thought that maybe all would be a big data set....but if you think we can run it on the whole database then we can...

Reply:
We have <500 entries in the DB, running it on all should be fine